Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN curl -L https://deb.nodesource.com/setup_10.x | bash - \
            apt-utils \
            build-essential \
            dirmngr \
+           dumb-init \
```

Comment:
?

Reply:
The latest Debian image from today deletes dumb-init. https://github.com/apache/airflow/pull/7278

Reply:
It's only the second commit that is relevant - this change is based on #7278 indeed. Seems that latest release of python images removed pre-installed dumb-init (there was a discussion in debian devlists about removing dumb-init at some point in time). We used it now so we have to install it and use the newly installed one.

Reply:
But now when I think about it - I think I will solve it differently. It's already quite some time when docker and docker-compose support --init flag which defaults to run "tini" as the init process. this will prevent us to rebuild the image and make the whole docker compose cleaner.